addon/components/select-input.js

```diff
@@ -30,6 +32,20 @@ export default AbstractInput.extend({
   // Computed Properties
   // ==========================================================================
 
+  @readOnly
+  @computed('store.formValue')
+  disabled (value) {
+    const modelDef = this.get('model')
+    const cellConfig = this.get('cellConfig')
+    const bunsenId = this.get('bunsenId')
```

Comment:
I think since the values of those properties effect the output of the computed property they should be included in the dependencies. In reality they shouldn't be constantly changing and probably won't effect the end result but if for some reason they are changing the disabled functionality should properly reflect that.